<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_TextureMesh</class>
 <widget class="QDialog" name="Dialog_TextureMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>纹理映射</string>
  </property>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseInputDir">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>纹理模型输出目录:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>三角网模型文件路径:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CANCEL">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseOutputDir">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OutputDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纹理参数设置:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inputDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>纹理模型输出格式:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ply</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>obj</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
